src/gdphelper/gdpdescribe.py

```diff
@@ -1,28 +1,146 @@
 import pandas as pd
 import numpy as np
 
-def gdpdescribe(df, x="GDP", y="year"):
+def gdpdescribe(df, x, y, stats=["mean", "sd", "median"], dec=2):
     """
-    calculates summary statistics for numeric variable x by categorical variable y
+    Calculates summary statistics for the Numeric Variable X grouping by categorical variable Y.

```

Comment:
```suggestion
    Calculates summary statistics for the Numeric Variable x grouping by categorical variable y.

```

To match the argument specified :)